bitnami/contour/Chart.yaml

```
@@ -32,4 +32,4 @@ maintainers:
 name: contour
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/contour
-version: 19.0.3
+version: 19.0.4
```

As we are modifying the behavior of the deployment, please bump the chart to a new minor

```suggestion
version: 19.1.0
```